INSTALL.rst

```diff
@@ -1,64 +1,509 @@
-instructions on `venv`_ and `virtual environments`_.
+your computer and you intend to install ``scikit-image`` inside of it.
+
+We also make a few more assumptions about your system:
+
+- You have a C compiler setup.
+- You have a C++ compiler setup.
+- You are running a version of python compatible with our system as listed
+  in our `setup.py file <https://github.com/scikit-image/scikit-image/blob/master/setup.py#L212>`_.
+- You've cloned the git repository into a directory called ``scikit-image``. 
+  This directory contains the following files:
+
+.. code-block::
+
+    scikit-image
+    ├── asv.conf.json
+    ├── azure-pipelines.yml
+    ├── benchmarks
+    ├── CODE_OF_CONDUCT.md
+    ├── CONTRIBUTING.txt
+    ├── CONTRIBUTORS.txt
+    ├── doc
+    ├── INSTALL.rst
+    ├── LICENSE.txt
+    ├── Makefile
+    ├── MANIFEST.in
+    ├── README.md
+    ├── RELEASE.txt
+    ├── requirements
+    ├── requirements.txt
+    ├── setup.cfg
+    ├── setup.py
+    ├── skimage
+    ├── TODO.txt
+    ├── tools
+    └── viewer_examples
+
+All commands below are assumed to be running from the ``scikit-image``
+directory containing the files above.
+
+
+Build environment setup
+-----------------------
+
+Once you've cloned your fork of the scikit-image repository,
+you should set up a Python development environment tailored for scikit-image.
+You may choose the environment manager of your choice.
+Here we provide instructions for two popular environment managers:
+``venv`` (pip based) and ``conda`` (Anaconda or Miniconda).
+
+venv
+====
+When using ``venv``, you may find the following bash commands useful:
+
+.. code-block:: sh
+
+  # Create a virtualenv named ``skimage-dev`` that lives in the directory of
+  # the same name
+  python -m venv skimage-dev
+  # Activate it. On Linux and MacOS:
+  source skimage-dev/bin/activate
+  # Install all development and runtime dependencies of scikit-image
+  pip install -r <(cat requirements/*.txt)
+  # Build and install scikit-image from source
+  pip install -e . -vv
+  # Test your installation
+  pytest skimage
+
+On Windows, please use ``skimage-dev\Scripts\activate`` on the activation step.
+
+conda
+=====
+
+When using conda for development, we
+recommend adding the conda-forge channel for the most updated version
+of many dependencies.
+Some dependencies we use (for testing and documentation) are not available
+from the default anaconda channel. Please follow the official
+`conda-forge installation instructions <https://conda-forge.org/#about>`_
+before you get started.
+
+.. code-block:: sh
+
+  # Create a conda environment named ``skimage-dev``
+  conda create --name skimage-dev
+  # Activate it
+  conda activate skimage-dev
+  # Install major development and runtime dependencies of scikit-image
+  # (the rest can be installed from conda-forge or pip, if needed)
+  conda install `for i in requirements/{default,build,test}.txt; do echo -n " --file $i "; done`
```

Comment:
Ha, love this.

Reply:
the problem is that it doesn't work with windows.